.github/workflows/test.yml

```diff
@@ -40,8 +40,8 @@ jobs:
           yarn compile:l1
           yarn compile:l2
       
-      - name: Run tests
+      - name: Run tests starknet
         run: |
           yarn testnet:l2 &
-          yarn testnet:l1 &
-          yarn test
\ No newline at end of file
+          sleep 30
```

Comment:
why `sleep 30`?

Reply:
starknet-devnet node crash sometimes

Reply:
don't forget to compile contracts before running your test. you can also split "devnet launch" and "tests run" in different steps, as it's done here: https://github.com/aave-starknet-project/aave-starknet-bridge/blob/main/.github/workflows/e2e-tests.yml#L58